_css/app.css

```diff
@@ -438,26 +437,41 @@ img.ecosystem-image {
   margin-right: 20px;
 }
 
-.jumbotron {
-  background-image: url('../assets/infra/splash-img.png');
+.page-heading {
+  color: #fff;
+  padding: 3rem;
+  background-image: url('https://raw.githubusercontent.com/cormullion/graphics/ed1bbef52ae2081a5a192f23d35a0cb9b38b8d30/svg-banners/backsplash-3.svg');
```

Comment:
Perhaps this should be copied across - the repo here is only temporary...

Reply:
I'm on it , PR incoming